src/app/components/TextEditorWrapper.tsx

```
@@ -88,6 +96,23 @@ type EditorDerivedState = {
+  icon: string;
+  title: string;
+  isActive: boolean;
+  onClick: () => void;
+}
+
+const FormatButton: React.FC<FormatButtonProps> = ({ icon, title, isActive, onClick }) => (
+  <button
+    className={`${styles.formatButton} ${isActive ? styles.active : ''}`}
+    onClick={onClick}
+    title={title}
+  >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)